packages/microcosm-www-next/src/pages/index.js

```diff
@@ -2,12 +2,32 @@ import React from 'react'
 import Link from 'gatsby-link'
 
 const IndexPage = () => (
-  <div>
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <Link to="/page-2/">Go to page 2</Link>
-  </div>
+  <section>
+    <div className="wrapper">
+      <h1>
+        <span>01.</span>
+        Rendering a View
+      </h1>
+
+      <div className="figure-container">
+        <figure className="figure -left">
```

Comment:
does it make sense to use `figure` here?

Reply:
I think so